delivery_purchase/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 Tecnativa - Carlos Roca
```

Comment:
This and the view should go into `wizards` directory, being a transient model. Odoo is not complying with this guideline though.